node/rpc-client/src/common_ops.rs

```diff
@@ -19,3 +19,7 @@ pub async fn shutdown(
 ) -> Result<ShutdownResult, Error> {
     call(SHUTDOWN, params, auth_token).await
 }
+
+pub async fn start_time(auth_token: &Option<String>) -> Result<StartTimeResult, Error> {
```

Comment:
Not a big deal but for automatic js binding to work we need to have two arguments for all of our endpoints:

So use prototype for XXX endpoint:
`(XXXParams, &Option<String>) -> Result<XXXResult, Error>`

Reply:
Added